sentry/const.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 Versada <https://versada.eu/>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import collections
+import logging
+
+import odoo.loglevels
+
+_logger = logging.getLogger(__name__)
+try:
+    import raven
+    from raven.conf import defaults
+except ImportError:
+    _logger.debug('Cannot import "raven". Please make sure it is installed.')
+
+
+def split_multiple(string, delimiter=',', strip_chars=None):
+    '''Splits :param:`string` and strips :param:`strip_chars` from values.'''
+    if not string:
+        return []
+    return [v.strip(strip_chars) for v in string.split(delimiter)]
+
+
+SentryOption = collections.namedtuple(
+    'SentryOption', ['key', 'default', 'converter'])
+
+# Mapping of Odoo logging level -> Python stdlib logging library log level.
+LOG_LEVEL_MAP = dict([
+    (getattr(odoo.loglevels, 'LOG_%s' % x), getattr(logging, x))
+    for x in ('CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET')
+])
+DEFAULT_LOG_LEVEL = 'warn'
+
+DEFAULT_TRANSPORT = 'threaded'
+TRANSPORT_CLASS_MAP = {
+    'requests_synchronous': raven.transport.RequestsHTTPTransport,
+    'requests_threaded': raven.transport.ThreadedRequestsHTTPTransport,
+    'synchronous': raven.transport.HTTPTransport,
+    'threaded': raven.transport.ThreadedHTTPTransport,
+}
+
+ODOO_USER_EXCEPTIONS = [
+    'odoo.exceptions.AccessDenied',
+    'odoo.exceptions.AccessError',
+    'odoo.exceptions.MissingError',
+    'odoo.exceptions.RedirectWarning',
+    'odoo.exceptions.UserError',
+    'odoo.exceptions.ValidationError',
+    'odoo.exceptions.Warning',
+    'odoo.exceptions.except_orm',
```

Comment:
After thinking for a bit (and drinking a bit of coffee), I think actually the better question is this:

What is the definition of a "User Error" in the context of this app, and how can they be changed?